src/en-GB/users/index.html

```diff
@@ -0,0 +1,120 @@
+        <h3 class="h3">Lorem ipsum</h3>
+        <p class="mb-8 lg:mb-12 p">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec in dui sed arcu ullamcorper rhoncus. Maecenas lacus ligula, aliquam
+          quis elit vitae, congue viverra libero.
+        </p>
+        <h3 class="h3">Lorem ipsum</h3>
+        <p class="mb-0 p">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec in dui sed arcu ullamcorper rhoncus. Maecenas lacus ligula, aliquam
+          quis elit vitae, congue viverra libero.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="bg-grey-300 section section--full">
+  <div class="wrapper">
+    <h2 class="h2 mb-12 lg:mb-16 text-center">Dive into Ceph</h2>
+    <ul class="grid lg:grid--cols-3 list-none m-0 p-0">
+      <li class="bg-white border-grey-500 border-px border-solid relative rounded-2">
+        <div class="grid grid--gap-0 md-to-lg:grid--cols-2 overflow-hidden rounded-2">
+          <div class="aspect-ratio aspect-ratio--16x9 aspect-ratio--cover md-to-lg:h-full tint tint--black-2">
+            <img alt="" class="absolute md-to-lg:h-full left-0 top-0" loading="lazy" src="/assets/bitmaps/photo-ink-01.jpg" />
+            <img alt="" class="absolute bottom-0 left-0 m-4 w-12 z-1" src="/assets/svgs/icon-documentation-white-red.svg" />
+          </div>
+          <div class="p-5">
+            <h3 class="h3">Documentation</h3>
+            <p class="p">
+              Find out everything you need to know about Ceph, including component, guides, coding and more, with our extensive
+              documentation library. From here you can get started with Ceph.
+            </p>
+            <a class="a link-cover link-cover--shadow" href="/{{ locale }}/users/documentation/">Get started with Ceph (documentation)</a>
+          </div>
+        </div>
+      </li>
+      <li class="bg-white border-grey-500 border-px border-solid relative rounded-2">
+        <div class="grid grid--gap-0 md-to-lg:grid--cols-2 overflow-hidden rounded-2">
+          <div class="aspect-ratio aspect-ratio--16x9 aspect-ratio--cover md-to-lg:h-full tint tint--black-2">
+            <img alt="" class="absolute md-to-lg:h-full left-0 top-0" loading="lazy" src="/assets/bitmaps/photo-ink-02.jpg" />
+            <img alt="" class="absolute bottom-0 left-0 m-4 w-12 z-1" src="/assets/svgs/icon-code-white-red.svg" />
+          </div>
+          <div class="p-5">
+            <h3 class="h3">Code</h3>
+            <p class="p">
+              Explore and create novel, innovative functions with the support and expertise of a global community. Whether you are new to
+              coding or a seasoned expert, input is always valuable.
+            </p>
+            <p class="p">Find out where Ceph is headed and where your coding skill could be applied.</p>
+            <a class="a link-cover link-cover--shadow" href="/{{ locale }}/developers/code/">Code Ceph</a>
```

Comment:
Similarly here, we'll want to confirm which sections/pages we want to highlight here on the Users hub, either from the Users section itself or other site sections

Reply:
Agreed. As more content flows in we can adjust the cross-linking as necessary.